lib/oli/delivery/metrics.ex

```diff
@@ -703,6 +862,33 @@ defmodule Oli.Delivery.Metrics do
     end)
   end
 
+  def proficiency_per_student_for_page(%Section{id: section_id, analytics_version: :v2}, page_id) do
+
+    page_type_id = Oli.Resources.ResourceType.get_id_by_type("page")
+
+    query =
+      from(summary in Oli.Analytics.Summary.ResourceSummary,
+        where: summary.section_id == ^section_id
+          and summary.project_id == -1
+          and summary.publication_id == -1
+          and summary.user_id != -1
+          and summary.resource_id == ^page_id
+          and summary.resource_type_id == ^page_type_id,
+        select:
+          {summary.user_id,
+           fragment(
+             "CAST(? as float) / CAST(? as float)",
```

Comment:
Because of the process in which we create these records, it isn't possble to have num_attempt_attempts = 0.  If there was no attempt for a scope, there wouldn't be a record